### ISSUE TITLE
Auto upload: upon new synced folder check for duplicate

### ISSUE THREAD

**Issue by jcklpe** [nextcloud/android#7318]
### Steps to reproduce
1. Turn on phone 
2. Phone connects to wifi
3. Phone sends message saying that there is an upload conflict 
4. Select "image already on phone" and hit submit
5. Phone sends another "upload conflict" message and it's for the exact same picture 

### Expected behaviour
- it should just use the image already uploaded as selected rather than continuously conflict. 

### Actual behaviour
- it just loops back over. 

### Environment data
Android version: android 11

Device model: pixel 2

Stock or customized system:

Nextcloud app version: 3.14 rc

Nextcloud server version: 20

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by jcklpe** [comment]
![Screenshot_20201113-145849](https://user-images.githubusercontent.com/9451726/99121066-8ed26100-25c1-11eb-8463-e2a1c83905ba.png)

**Comment by tobiasKaminsky** [comment]
Duplicate of #6938

**Comment by jcklpe** [comment]
@tobiasKaminsky  the duplicate story does not match my issue. My issue is not that it asks me to make a decision on overwriting something. My issue is that when it asks me, I tell it to overwrite and then it asks me that same thing again over and over no matter if I tell it to overwrite or not.

**Comment by jcklpe** [comment]
Btw I think I solved the issue.

I had a sync folder for DCIM/camera and then when a new folder was set up for DCIM/Facebook, nextcloud created a sync task for DCIM/. Having the camera and the parent folder both syncing was leading to the conflict I believe. I disabled the camera folder sync and so far I haven't had a repeat of the issue. 

This does seem like a mistake thst nextcloud should take into account and cross check for.

**Comment by tobiasKaminsky** [comment]
Wow.
Never thought about this…

So we need to check if a new synced folder is a subset or parent of any other synced folder.

**Comment by uusijani** [comment]
Previously reported in #4817.

**Comment by jcklpe** [comment]
I  having the se problem again now with a new file and it doesn't seem to be a result of the nested folder issue.

What I do notice with this new issue is that the date for images is 1969. Which obviously these images are not both from 1969 but I believe that's the start of Unix time, so I think this has something to do with the dates getting screwy? 

![Screenshot_20201128-231809](https://user-images.githubusercontent.com/9451726/100534024-6f961f00-31d0-11eb-8517-c6f3e56fdaff.png)

**Comment by tobiasKaminsky** [comment]
@jcklpe this will be fixed with next 3.14.1, coming this week.

**Comment by AndyScherzinger** [comment]
Released
